src/Controls/src/Core/AnimationExtensions.cs

```diff
@@ -260,8 +273,24 @@ static void AnimateInternal<T>(IAnimatable self, IAnimationManager animationMana
 
 			s_animations[key] = info;
 
+			if (self is VisualElement ve)
+			{
+				ve.Unloaded += OnVisualElementUnloaded;
```

Comment:
I'm curious if there's a different place we can use to remove the animation then wiring to unloaded